docs/Training-PPO.md

```diff
@@ -99,7 +111,23 @@ the action is a very complex interaction between the observation variables, this
 
 Typical Range: `32` - `512`
 
-### Training Statistics
+### (Optional) Recurrent Neural Network Hyperparameters
+
+The below hyperparameters are only used when `use_recurrent` is set to true.
+
+#### Sequence Lenght
```

Comment:
Length

Reply:
Gah! I even put it into a google doc to check for spelling, and it didn't flag this. Good catch.